tests/catboost/test_catboost_model_export.py

```diff
@@ -229,7 +229,7 @@ def test_log_model(cb_model, tmpdir):
         model_config = Model.load(os.path.join(local_path, "MLmodel"))
         assert pyfunc.FLAVOR_NAME in model_config.flavors
         assert pyfunc.ENV in model_config.flavors[pyfunc.FLAVOR_NAME]
-        env_path = model_config.flavors[pyfunc.FLAVOR_NAME][pyfunc.ENV]
+        env_path = model_config.flavors[pyfunc.FLAVOR_NAME][pyfunc.ENV]["conda"]
```

Comment:
Can we verify that virtualenv flavor conf points to the correct file throughout these model flavor test cases as well?

Reply:
Sure, I'll add checks to verify that.